### PR TITLE
chore: include log validationNrql

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: recipes/newrelic/logs/logs.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ keywords:
 processMatch:
   - /logs/
 
+validationNrql: "SELECT count(*) from Log where hostname like '{{.HOSTNAME}}' SINCE 10 minutes ago"
```
Comment (Julien4218) [inline recipes/newrelic/logs/logs.yml:17]: We found out sometimes (for aws linux2), hostname is being replaced by short form, but the reported host name can be long form. So if you just add a '%' after the 2 closing brackets, that takes care of the issue.

**Review by Julien4218 — CHANGES_REQUESTED** [review]
Just need a wildcard character after hostname

File: recipes/newrelic/logs/logs.yml (hunk 1 above)
Comment (ctrombley) [reply recipes/newrelic/logs/logs.yml:17]: Hopefully in the future we can switch over to agent/entity ID for a more precise validation.  If this works for now then :shipit:

**Review by ctrombley — APPROVED** [review]

File: recipes/newrelic/logs/logs.yml (hunk 1 above)
Comment (Julien4218) [reply recipes/newrelic/logs/logs.yml:17]: Would you like me to make the change?

**Pull request merged**